src/TestInfrastructure/Helpers/MefTestHelpers.cs

```diff
@@ -74,7 +74,7 @@ public static Export CreateExport<T>(object exportInstance, IDictionary<string,
         /// <param name="optionalExports">Any optional exports to include in the composition.</param>
         /// <param name="requiredExports">Any exports required by TTypeToCheck. Optional.</param>
         public static void CheckTypeCanBeImported<TTypeToCheck, TImportType>(
-            object importer,
+            SingleObjectImporter<TImportType> importer,
```

Comment:
From memory, this test method is supposed to be entirely generic - you should be able to able to pass it anything it will check that supplied instance can import the specified types.
The SingleObjectImporter<> is just a helper class that helps test that a particular type can be MEF imported.